corehq/apps/hqadmin/management/commands/republish_couch_case_changes.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management import BaseCommand
+
+from dimagi.utils.chunked import chunked
+
+from casexml.apps.case.models import CommCareCase
+from corehq.doctypemigrations.continuous_migrate import _bulk_get_revs
```

Comment:
should this be renamed to `bulk_get_revs` if it is used in other places?

Reply:
Yes, updated.